### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Calandiel**
Beep. Just checking if Github app works

### CODE REVIEW

**Comment by jjonj** [comment]
This looks terrible... you're fired.

**Comment by Calandiel** [comment]
:c
The army of Angmar shall find u :c

**Comment by jjonj** [comment]
I am Melkor of Angban, so bring it.

**Comment by jjonj** [comment]
Melkor = Morgoth = The master of Sauron.
Angban = place waaay up north, not isengard

**Comment by Moopli** [comment]
I like it, if no objections I can merge

File: Angmar inquisition.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is where we burn people
```
Comment (jjonj) [inline Angmar inquisition.txt]: Salemn? Please correct this.

**Comment by Calandiel** [comment]
You have no power in the land of Eru, Melkor! I Ca' nauva sinome e' inyesse!

**Comment by jjonj** [comment]
Lle na elen feuya edan

**Comment by Calandiel** [comment]
it doesnt look like quenya with intervocal "d" :c

**Comment by jjonj** [comment]
Sorry I'm a bit rusty.
Meleamin

**Comment by Calandiel** [comment]
Better. You got a plus: +1

**Comment by klay2000** [comment]
elvish?

**Comment by Calandiel** [comment]
Mae. Or "mai". Or some other versions. Depending on reconstruction/era. To sum up: yeah

**Comment by klay2000** [comment]
I have a friend who speaks it :)

**Comment by T0N12** [comment]
Should we delete this merge request?

**Comment by jjonj** [comment]
Yeah why not

**Pull request closed without merging**